CITATION.cff

```diff
@@ -0,0 +1,23 @@
+# Parser settings.
+cff-version: 1.2.0
+message: Please cite this crate using these information.
```

Comment:
Another required information is a message for citation.  Usually, it asks recipients politely to cite the project using the given information.  I entered this as an initial value but I can also change it, if required.